docs/general/accessibility.md

```diff
@@ -0,0 +1,39 @@
+# Accessible Charts
+
+Chart.js charts are rendered on user provided `canvas` elements. Thus it is up to the user to create the `canvas` element in a way that is accessible. The `canvas` element has support in all browsers and will render on screen but the `canvas` content will not be accessible to screen readers.
+
+With `canvas`, the accessibility has to be added with JavaScript for `ARIA` attributes on the `canvas` element or added using internal fallback content placed within the opening and closing canvas tag.
```

Comment:
remove "JavaScript for " ( you can potentially create ARIA attributes without JavaScript)